cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -120,3 +134,12 @@ func (cmd *command) validateFlags() error {
 
 	return nil
 }
+
+func (cmd *command) getLogFunc() func(format string, v ...interface{}) {
```

Comment:
Would it be possible to extract `getLogFunc()` to be reused by both `deploy` and `uninstall` commands?